bin/hive_setup

```diff
@@ -149,11 +234,65 @@ def setup_hive(dir, mods)
     end
   end
 
+  puts '=========================='
+  print "Setting ruby ... "
+  File.open("#{dir}/.ruby-version", 'w') do |rv|
+    rv.write ruby
+  end
+  print "SUCCESS\n"
+
+  print "Creating daemon startup script..."
+  if os == 'Linux'
+    startup_script = "#{dir}/hived.init"
+    config_file = "#{dir}/hived.conf"
+    write_logs = '> $LOG_DIRECTORY/hived.out 2> $LOG_DIRECTORY/hived.err'
+    File.open("#{dir}/start_hive.sh", 'w', 0755) do |f|
+      f.write "#!/bin/bash -l\n"
```

Comment:
For the Android runner, 'bash -li' is required so that the environment variables required for adb are set